tools/interactive/interactivetool_jupyter_notebook.xml

```diff
@@ -17,12 +17,22 @@
     </environment_variables>
     <command detect_errors="aggressive"><![CDATA[
         #import re
+        #import random
+        #import string
         export GALAXY_WORKING_DIR=`pwd` &&
         mkdir -p ./jupyter/outputs/ &&
         mkdir -p ./jupyter/data &&
 
-        #set $cleaned_name = re.sub('[^\w\-\.]', '_', str($input.element_identifier))
-        ln -sf '$input' './jupyter/data/${cleaned_name}' &&
+        #set $cleaned_names = []
+        #for $file in $input
+            #set $cleaned_name = re.sub('[^\w\-\.]', '_', str($file.element_identifier))
```

Comment:
```suggestion
            #set $cleaned_name = re.sub('[^\w\-\.\s]', '_', str($file.element_identifier))
```
Since we're running the container we can be pretty sure whitespace is ok

Reply:
Cool, I've included this also in the notebook's name when running on existing